config/settings.py

```diff
@@ -131,9 +134,9 @@
 
 STATIC_URL = "static/"
 
-STATIC_ROOT = PROJECT_DIR / "staticfiles"
+STATIC_ROOT = BASE_DIR / "staticfiles"
 
-# STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
+STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
```

Comment:
not sure if you have all here, I have:

```
STATIC_HOST = config('DJANGO_STATIC_HOST', default='')
STATIC_URL = STATIC_HOST + '/static/'

STATIC_ROOT = os.path.join(PROJECT_ROOT, 'staticfiles')
STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'

STATICFILES_DIRS = (
    os.path.join(PROJECT_ROOT, 'static'),
)
```

At the top of settings.py I have this variable set as well:
PROJECT_ROOT = os.path.dirname(os.path.abspath(__file__))

Reply:
I think some of that does not apply to my folder structure, but it is similar.

Reply:
Good point, now I see your `static` folder sits at the root of the project, I recommend moving it into the project folder so `config` in your case, then these settings should work.